### PR TITLE
[MIG] fetchmail_thread_default: Migration to 12.0

### PR DESCRIPTION
**Pull request by Rad0van**
Straightforward port to 12.0

### CODE REVIEW

**Comment by Rad0van** [comment]
@chienandalu, @sergio-teruel, @Yajo could you have a look please? Thank you.

File: .vscode/settings.json
Comment (sergio-teruel) [inline .vscode/settings.json]: Remove this file

File: fetchmail_thread_default/README.rst
Comment (sergio-teruel) [inline fetchmail_thread_default/README.rst]: Use oca-gen-addon-readme OCA utility to regerate this file.
https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-12.0#tasks-to-do-in-the-migration

**Review by sergio-teruel — APPROVED** [review]
Code review, little comments

File: fetchmail_thread_default/README.rst
Comment (Rad0van) [reply fetchmail_thread_default/README.rst]: Done.

**Comment by Rad0van** [comment]
> Code review, little comments

@sergio-teruel, after fighting with squashing commits I hope I got it right. Please check.

**Comment by sergio-teruel** [comment]
Ok for me... :+1:

**Comment by Rad0van** [comment]
> Ok for me... 👍

@sergio-teruel thanx. As soon as it gets merged I plan to migrate to 13.0 as that is in fact the version I need ;-)

**Review by yajo — APPROVED** [review]

**Pull request merged**
